.env

```diff
@@ -0,0 +1,10 @@
+DSN="mysql:host=localhost;dbname=local;unix_socket=/Users/tarecord/Library/Application Support/Local/run/49J5D1kk2/mysql/mysqld.sock;"
+DB_HOST=localhost
+DB_PORT=3306
+DB_NAME=local
+DB_USER=root
+DB_PASSWORD=root
+WP_URL=https://sign-in-with-google.dev
+TABLE_PREFIX=wp_
+ADMIN_USERNAME=tarecord
+ADMIN_PASSWORD=b8Ewroaw
```

Comment:
Did you mean to commit this file?  I think a sample/starter .env would be good but this is showing a lot of your local info.

Reply:
I didn't mean to commit this, I had it from another branch and didn't notice 🤦🏼‍♂️ I'll convert it to a `.env.example`